BuilderDesignPattern/Program.cs

```diff
@@ -0,0 +1,411 @@
+            //IPizzaBuilder pizzaBuilder;
+            //Director director;
+            //Exception invalidInputException = new Exception("Invalid input");
+            //string input;
+
+            //string dough = "";
+            //string sauce = "";
+            //List<string> topping = new List<string>();
+
+            //Console.WriteLine("Welcome to the pizza restaurant");
+            //Console.WriteLine("---------------------------------");
+
+            //Console.Write("What kind of pizza do you prefer? (Enter item number)");
+            //Console.Write("\n");
+            //Console.Write("1. White Pizza");
+            //Console.Write("\n");
+            //Console.Write("2. Wheat Pizza");
+            //Console.Write("\n");
+
+            //pizzaBuilder = Console.ReadLine() switch
+            //{
+            //    "1" => new WhitePizzeBuilder()
+            //    ,
+            //    "2" => new WheatPizzaBuilder()
+            //    ,
+            //    _ => throw invalidInputException
+            //};
+
+            //director = new Director(pizzaBuilder);
+            //dough = pizzaBuilder is WhitePizzeBuilder ? "White" : "Wheat";
+
+            //// sauce
+            //Console.Write("\n");
+            //Console.Write("What kind of sauce? (Enter item number)");
+            //Console.Write("\n");
+            //Console.Write("1. Red sauce");
+            //Console.Write("\n");
+            //Console.Write("2. White sauce");
+            //Console.Write("\n");
+            //Console.Write("3. No sauce");
+            //Console.Write("\n");
+
+            //string sauceType = Console.ReadLine() switch
+            //{
+            //    "1" => "Red"
+            //    ,
+            //    "2" => "White"
+            //    ,
+            //    "3" => null
+            //    ,
+            //    _ => throw invalidInputException
+            //};
+            //sauce = sauceType;
+
+
+            //// topping
+            //Console.Write("\n");
+            //Console.Write("What about topping? (Enter item number)");
+            //Console.Write("\n");
+            //Console.Write("1. Gorgonzola");
+            //Console.Write("\n");
+            //Console.Write("2. Mushroom");
+            //Console.Write("\n");
+            //Console.Write("3. Broccoli");
+            //Console.Write("\n");
+            //Console.Write("4. That's it thanks");
+            //Console.Write("\n");
+
+            //while ((input = Console.ReadLine()) != "4")
+            //{
+            //    topping.Add(
+            //        input switch
+            //        {
+            //            "1" => "Gorgonzola"
+            //            ,
+            //            "2" => "Mushroom"
+            //            ,
+            //            "3" => "Broccoli"
+            //            ,
+            //            _ => throw invalidInputException
+            //        }
+            //        );
+
+            //    Console.Write("and: ");
```

Comment:
It was really great asking for extra toppings using and,, you have used friendly language for entering the order! Thank you 👏